cpp/cmake/Dependencies.cmake

```diff
@@ -39,7 +39,7 @@ else(DEFINED ENV{RAFT_PATH})
 
   ExternalProject_Add(raft
     GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           196f8c9aa52a22e326a5ef8e73047d13497a5c69
+    GIT_TAG           c5030a0f9cc7f95ce4208febc89b77ccf251b60e
```

Comment:
This file no longer exists now that #3844 has been merged and this change is no longer needed since now we pull `branch-0.21` Instead of a pinned commit by default.